helpers/securewebhook/securewebhook.go

```diff
@@ -27,3 +38,25 @@ func GetRequestBody(s *Settings) ([]byte, error) {
 	}
 	return b, nil
 }
+
+// VerifySignature uses the ECDSA publicKey and, verifies received payload and signature
+func VerifySignature(publicKey *ecdsa.PublicKey, payload []byte, signature, timestamp string) (bool, error) {
```

Comment:
Would the timestamp be a request header?

Reply:
correct

This brings up a good idea to have some constants for these header names. As a consumer of SDKs, I like having enums and the likes instead of me searching the docs for what these header names might be and creating those myself.

```const (
	VerificationHTTPHeader = "X-Twilio-Email-Event-Webhook-Signature"
	TimestampHTTPHeader = "X-Twilio-Email-Event-Webhook-Timestamp"
)
```

Reply:
what do you mean? Do you want the header to be parsed in the function?

Reply:
Add these variables at the package level.